apps/aehttp/test/aehttp_sc_SUITE.erl

```diff
@@ -794,6 +798,10 @@ sc_ws_open_(Config, ChannelOpts0, MinBlocksToMine, LogDir) ->
             ok = ?WS:unregister_test_for_channel_events(IConnPid, TestEvents),
             ok = ?WS:unregister_test_for_channel_events(RConnPid, TestEvents)
     end,
+
+    CountAfter = get_channels_count(),
+    ct:log("CountAfter = ~p", [CountAfter]),
+
```

Comment:
Maybe check something w.r.t.`CountBefore` and `CountAfter`?

Reply:
I do that in the `aesc_fsm_SUITE`. For some reason, when I did that in the `aehttp_sc_SUITE`, the test run got stuck.